packages/wdio-allure-reporter/src/index.ts

```diff
@@ -126,6 +127,11 @@ class AllureReporter extends WDIOReporter {
     }
 
     onTestStart(test: TestStats | HookStats) {
+        let testObj:any =test
```

Comment:
Don't use `any`. If you want access `TestStats.argument` add in `TestStats` type

Reply:
This was throwin hookStats doesn't contain arguments error when i `run npm run compile` thats why i provided any

Reply:
Seems here we need some refactoring. Btw I don't see reason to hack typescript types in such a way by using `any`

Reply:
@BorisOsipov  should i add argument to hookstat interface ?

Reply:
Not sure. I will be easy to add type guards like
```
        if (test instanceof TestStats) {
            const dataTable = test.argument ? test.argument.rows.map((a:{cells:null}) => a.cells) : undefined
            dataTable ? stringify(dataTable, function (err, records) {
                records ? AllureReporter.addAttachment('Data Table', records, 'text/csv') : console.log(err)}) : null
        }
```

and the same for `argument.rows`

Reply:
@BorisOsipov  i used :

```
    onTestStart(test: TestStats | HookStats) {
        const testObj = test as TestStats
        const argument = testObj.argument as Argument
        const dataTable = testObj.argument ? argument.rows.map((a: { cells: null }) => a.cells) : undefined
        if (dataTable) this.addAttachment({ name: 'Data Table', content: stringify(dataTable), type: 'text/csv' })
        const testTitle = test.currentTest ? test.currentTest : test.title
```

Reply:
ok